miasm2/expression/expression.py

```diff
@@ -318,12 +318,10 @@ def replace_expr(self, dct=None):
         @dct: dictionary of Expr -> *
         """
         if dct is None:
-            dct = {}
+            return self
```

Comment:
Good point. While you are here, you might want to rewrite the test above as:
```Python
        if not dct:
            return self
```
(since there is no reason to do something either when `dct` is empty).